### PR TITLE
Attempting to clarify that negative/inverse versions of a title repre…

### PR DESCRIPTION
**Pull request by bigokro**
…sent a functional difference

rather than just an alternative version of the text.

### CODE REVIEW

File: README.mediawiki [hunk 1]
```diff
@@ -639,9 +639,9 @@ A Claim is a Dialectic, and yet it must be stated in the form of some type of af
...
+* '''Variations in emotional "temperature"''' - It might be interesting to view other versions of a Claim Title according to the emotional impact that the title might contain, independent of the actual content of the debate itself. One of the main problems with a typical Argument Map, especially one that seeks to separate the arguments from the individuals making the claims, is that the resulting map can be logical to the point of alienating the common reader. Following the principle that [[#227-we-should-inspire-with-emotion-but-argue-with-reason|"We should inspire with emotion, but argue with reason"]], it might be "inspiring" to see a debate that says "Only an idiot would believe that the Earth is flat", at least at first. Certainly users would find it entertaining to play with the "tempurature level" of debates as they explore the content of the Canonical Debate. Each of the types above (positive, negative and inquisitive) would maintain its own collection of variations on the title.
```
Comment (Whathecode) [inline README.mediawiki]: Typo in "tempurature level".

**Comment by Whathecode** [comment]
In the "changing perspectives" section, I would also expect ...

> This requires tools that permit actually choosing a perspective from which to view a debate.

... to state:

> ... from which to view *or link to* a debate.

It is this impact on 'linking to' nature of "positive and negative assertions" which makes it stand out significantly from other variations in the title. You cannot sensibly link a supporting statement to the inquisitive form of a statement, you will always have to specify whether you are adding a supporting statement to the positive or negative assertion.

Therefore, I still think 'negative title' belongs more as an attribute than a variation in the title. Some for the inquisitive form. The 'variations in emotional temperature' seem to live in a different dimension, orthogonal to these.

E.g., Does god exist? Do people still believe that god exists? (more emotion)

**Comment by Whathecode** [comment]
Another point which is also not recognized in the current paper: whether to show/use the positive or negative form of an assertion when making an argument depends on the argument being made. This is highly contextual, and not just a matter of 'which outcome is move favored' or how offended people are by a phrasing.

A good example of this are the following two statements: "Climate change is man-made" and "Global warming is man-made".

Climate change deniers are likely to want to add "Global warming is not man-made" as an opposing statement, whereas others will want to add "Global warming is man-made" as a supporting statement. **The same statement can thus appear on both sides in opposite forms.**

**Comment by bigokro** [comment]
The remaining points you bring up will be handled separately from this PR. Thanks!

**Pull request merged**